README.md

```diff
@@ -189,8 +210,8 @@ script_hash15hx806zf0g8kcv399dpxf6fq4l98myqpvvzj2rltg465uz36435
 
   $ cat addr.prv \
   | cardano-address key public --with-chain-code \
-  | cardano-address address payment --from-key --network-tag testnet \
-  | cardano-address address delegation --from-script $(cardano-address script hash "$(cat script.txt)")
+  | cardano-address address payment --network-tag testnet \
+  | cardano-address address delegation $(cardano-address script hash "$(cat script.txt)")
```

Comment:
This one actually does not work:

```
$ cat addr.prv \
  | cardano-address key public --with-chain-code \
  | cardano-address address payment --network-tag testnet \
  | cardano-address address delegation $(cardano-address script hash "$(cat script.txt)")
  
Invalid human-readable prefix. Prefix ought to be one of: ["stake_xvk"]

Usage: cardano-address address delegation (XPUB | SCRIPT HASH)
  Create a delegation address
```

I suppose the problem is in the `cardano-address address delegation` which doesn't want to accept script hash, although the message suggests `Usage: cardano-address address delegation (XPUB | SCRIPT HASH)
`:

```
$ cardano-address script hash "$(cat script.txt)" | tee script.hash
script1qd46g66dq98dh8h2qh947wq3qajf3plmlnkkedn2yexxke2lahd


$ cardano-address address delegation $(cat script.hash)
Invalid human-readable prefix. Prefix ought to be one of: ["stake_xvk"]

Usage: cardano-address address delegation (XPUB | SCRIPT HASH)
  Create a delegation address
```

Reply:
This one took me a while :o ... Apparently, we cannot combine multiple argument parsers with an alternative, they do not backtrack. So if the first one fails, the entire chain fails. That's a pity: 

https://github.com/pcapriotti/optparse-applicative/issues/166

Reply:
Also, I didn't caught this earlier because there was actually no test scenario for constructing a delegation address from a script. I've added one :+1: